packages/ui/src/hooks/use-social-sign-in-listener.ts

```diff
@@ -59,15 +59,15 @@ const useSocialSignInListener = () => {
       return;
     }
 
-    const { state, code, ...rest } = parseQueryParameters(window.location.search);
+    const { state, ...rest } = parseQueryParameters(window.location.search);
 
     if (!state || !stateValidation(state, parameters.connector)) {
       setToast(t('error.invalid_connector_auth'));
 
       return;
     }
 
-    void signInWithSocialHandler(parameters.connector, state, code ?? JSON.stringify(rest));
+    void signInWithSocialHandler(parameters.connector, JSON.stringify(rest));
```

Comment:
Need to rebase for [this PR](https://github.com/logto-io/logto/pull/969) changed this logic and merged into the master branch.

Reply:
already rebased and the change is intended. please check it again